ui/src/app/pages/profile/data-use-agreement-panel.tsx

```diff
@@ -0,0 +1,159 @@
+      My work may be logged, monitored, and audited by the <i>All of Us</i> Research Program to ensure
+      compliance with policies and procedures, as well as the demonstration project charges.
+    </InitialsAgreement>
+    <InitialsAgreement onChange={(v) => this.setState({initialName: v})} value={initialName}>
+      My name, affiliation, profile information, and research description will be made public.
+      My research description will be used by the <AoUTitle/> to provide
+      participants with meaningful information about the research being conducted.
+    </InitialsAgreement>
+    <InitialsAgreement onChange={(v) => this.setState({initialSanctions: v})}
+                       value={initialSanctions}>
+      Access granted by the program in accordance with this agreement is exclusively
+      for participation in <i>All of Us</i> demonstration projects. At the conclusion of my
+      participation in these activities, I understand that my access to the <i>All of Us</i> data
+      resources will be revoked and/or subject to customary access policies and procedures.
+    </InitialsAgreement>
+    <div><strong>I acknowledge that failure to comply with the terms of this agreement may result
+      in termination of my <AoUTitle/> account and/or other sanctions, including
+      but not limited to:</strong>
+      <IndentedUnorderedList>
+        <IndentedListItem>
+          <strong>posting my name and affiliation on a publicly-accessible list of violators, and</strong>
+        </IndentedListItem>
+        <IndentedListItem><strong>notifying the National Institutes of Health or other federal agencies of my
+          actions.</strong></IndentedListItem>
+      </IndentedUnorderedList>
+      <div><strong>I understand that failure to comply with these terms may also carry additional financial, legal, or other
+        repercussions.</strong>
+      </div>
+    </div>
+    <div>
+      {showSanctionModal &&
+      <SanctionModal onClose={() => this.setState({showSanctionModal: false})}/>}
+    </div>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
+                  disabled value={this.props.profileState.profile.username}/>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
+                  disabled value={this.props.profileState.profile.contactEmail}/>
+    <DuaTextInput style={{marginTop: '0.5rem'}}
+                  type='text' disabled value={new Date().toLocaleDateString()}/>
+    <TooltipTrigger content={errors && <div>
+      <div>All fields must be initialed</div>
+      <div>All initials must match</div>
+      <div>Initials must be six letters or fewer</div>
+    </div>}>
+      <Button
+        style={{marginTop: '1rem', cursor: errors && 'not-allowed', padding: '0 1.3rem'}}
+        disabled={errors || submitting} data-test-id='submit-dua-button'
+        onClick={() => this.submitDataUseAgreement(this.state.initialWork)}>Submit</Button>
```

Comment:
`submitDataUseAgreement` now takes `submitDataUseAgreement(dataUseAgreementVersion, initials)`

might need to merge in master to see this.

Reply:
this looks ok actually